msal/tests/dev apps/DesktopTestApp/MainForm.cs

```diff
@@ -469,8 +469,31 @@ private void clearLogsButton_Click(object sender, EventArgs e)
         private void authority_FocusLeave(object sender, EventArgs e)
         {
             _publicClientHandler.CreateOrUpdatePublicClientApp(this.authority.Text, publicClientId);
+        }

```

Comment:
Suggestion: Add a button to cancel the request to ensure we have easy e2e access to validate the functionality

Reply:
done and tested